examples/tpu/tpuvm_mnist.yaml

```diff
@@ -20,6 +20,7 @@ setup: |
     pip install "jax[tpu]>=0.2.16" -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
     pip install --upgrade clu
     pip install -e flax
+    pip install tensorflow tensorflow-datasets
```

Comment:
Why does `test_tpu_vm` work without this before?

Reply:
good question... it's because the latest version of `clu` package doesn't depend on `tensorflow` and `tensorflow-datasets` anymore. See: https://github.com/google/CommonLoopUtils/commit/390be7fd9d0e44f346a86d09d51477589ba9f19e